README.md

```diff
@@ -5,8 +5,6 @@
   By <a href="https://protofire.io/">Protofire</a>
 </p>
 
-[![Donate with Ethereum](https://en.cryptobadges.io/badge/micro/0xe8cdf02efd8ab0a490d7b2cb13553389c9bc932e)](https://en.cryptobadges.io/donate/0xe8cdf02efd8ab0a490d7b2cb13553389c9bc932e)
```

Comment:
I wouldn't remove this. While it's also at the bottom of the readme, that's a different context (the badges at the top of the readme vs. that specific section).